src/components/Books.js

```diff
@@ -1,20 +1,28 @@
 import './styles/Books.css';
+import { useSelector } from 'react-redux';
 import Book from './Book';
 import AddBook from './AddBook';
 
-const Books = () => (
-  <div className="books">
-    <ul>
-      <li className="book">
-        <Book title="Dating in the 21st century" author="Norman Aine" category="Love" />
-      </li>
-      <li className="book">
-        <Book title="Basic Computer Programming" author="Membe Joshua" category="Computer Science" />
-      </li>
-    </ul>
-    <hr className="line" />
-    <AddBook />
-  </div>
-);
+const Books = () => {
+  const bookList = useSelector((state) => state.books);
+  return (
+    <div className="books">
+      {bookList.map((b) => (
+        // eslint-disable-next-line react/jsx-key
+        <ul>
+          <li className="book">
+            <Book
+              title={b.title}
+              author={b.author}
+              id={b.id}
+            />
```

Comment:
- Optional
Kindly add a key property to each book. Currently, this error is showing up in the console.
![image](https://user-images.githubusercontent.com/33268927/201325226-c2a6032f-4d84-42a4-9bf9-42f37f6d862a.png)

Reply:
Thank you very much @muneebulrehman